### PR TITLE
PFW-1412: Update translations

### PR DESCRIPTION
**Pull request by 3d-gussner**
Fix button msg
Update internal translations
Update Dutch `F. jam detect`
Update pot and po files

- [ ] cherry-pick for MK3.12

### CODE REVIEW

File: lang/po/Firmware_cs.po [hunk 1]
```diff
@@ -206,7 +206,7 @@ msgstr "Podsviceni"
 #. MSG_TITLE_COMMUNICATION_ERROR c=20
 #: ../../Firmware/mmu2/errors_list.h:147 ../../Firmware/mmu2/errors_list.h:185
 msgid "COMMUNICATION ERROR"
-msgstr ""
+msgstr "CHYBA KOMUNIKACE"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po:209]: do we really want the MMU messages to be present in 3.12?

File: lang/po/Firmware_cs.po [hunk 2]
```diff
@@ -935,7 +946,7 @@ msgstr ""
 #. MSG_IMPROVE_BED_OFFSET_AND_SKEW_LINE1 c=20 r=4
 #: ../../Firmware/mesh_bed_calibration.cpp:2530
 msgid "Improving bed calibration point"
-msgstr ""
+msgstr "Zlepsovani kalibracniho bodu"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po]: Upresnovani

File: lang/po/Firmware_cs.po [hunk 3]
```diff
@@ -911,17 +922,17 @@ msgstr "Nyni provedu z kalibraci."
...
 #: ../../Firmware/mmu2/errors_list.h:126 ../../Firmware/mmu2/errors_list.h:165
 msgid "IDLER CANNOT MOVE"
-msgstr ""
+msgstr "CHYBA POHYBU ILDERU"
 
 #. MSG_TITLE_INVALID_TOOL c=20
 #: ../../Firmware/mmu2/errors_list.h:149 ../../Firmware/mmu2/errors_list.h:187
 msgid "INVALID TOOL"
-msgstr ""
+msgstr "NEPLATNY NASTROJ"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po:935]: neplatny slot filamentu

File: lang/po/Firmware_cs.po [hunk 4]
```diff
@@ -911,17 +922,17 @@ msgstr "Nyni provedu z kalibraci."
...
 #: ../../Firmware/mmu2/errors_list.h:127 ../../Firmware/mmu2/errors_list.h:164
 msgid "IDLER CANNOT HOME"
-msgstr ""
+msgstr "IDLER: CHYBA HOMINGU"
 
 #. MSG_TITLE_IDLER_CANNOT_MOVE c=20
 #: ../../Firmware/mmu2/errors_list.h:126 ../../Firmware/mmu2/errors_list.h:165
 msgid "IDLER CANNOT MOVE"
-msgstr ""
+msgstr "CHYBA POHYBU ILDERU"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po]: typo: ILDERU -> IDLERU

File: lang/po/Firmware_cs.po (hunk 1 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po:209]: This PR is for MK3 including the MMU2 messages. As mentioned above a cherry-pick to MK3_3.12 will be done after merge.

File: lang/po/Firmware_cs.po [hunk 5]
```diff
@@ -1678,22 +1701,24 @@ msgstr ""
 #: ../../Firmware/mmu2/errors_list.h:200 ../../Firmware/mmu2/errors_list.h:240
 msgid "Pulley motor stalled. Ensure the pulley can move and check the wiring."
 msgstr ""
+"Motor idleru se zastavil. Ujistete se, ze neni pohyb blokovan a zkontrolujte "
```
Comment (3d-gussner) [inline lang/po/Firmware_cs.po]: @DRracer is it Kladka motor?

File: lang/po/Firmware_cs.po [hunk 6]
```diff
@@ -2216,13 +2250,13 @@ msgstr "Vysouvam filament"
...
 msgid "Unloading to FINDA"
-msgstr ""
+msgstr "Vysunuti k FINDA"
 
 #. MSG_PROGRESS_UNLOAD_PULLEY c=20
 #: ../../Firmware/mmu2_progress_converter.cpp:12
 #: ../../Firmware/mmu2_progress_converter.cpp:38
 msgid "Unloading to pulley"
-msgstr ""
+msgstr "Vysunuti k idleru"
```
Comment (3d-gussner) [inline lang/po/Firmware_cs.po]: @DRracer again copy/paste issue

**Review by 3d-gussner — COMMENTED** [review]
pulley not idleru

File: lang/po/Firmware_cs.po (hunk 3 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po:935]: too long limited to c=20

File: lang/po/Firmware_cs.po (hunk 2 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po]: changing to `Upresnovani kalibracniho bodu`

File: lang/po/Firmware_cs.po [hunk 7]
```diff
@@ -1422,13 +1445,13 @@ msgstr ""
...
 msgid "PULLEY CANNOT MOVE"
-msgstr ""
+msgstr "KLADKA SE NEOTACI"
 
 #. MSG_PROGRESS_PARK_SELECTOR c=20
 #: ../../Firmware/mmu2_progress_converter.cpp:26
 #: ../../Firmware/mmu2_progress_converter.cpp:57
 msgid "Parking selector"
-msgstr ""
+msgstr "KLADKA SE NEOTACI"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po:1454]: :thinking: how did this happen? In the translation tool there is "Parkovani selektoru"

**Review by DRracer — APPROVED** [review]
Technically we can merge this PR for 3.12, but more fixes for the CZ translation are coming in near future.

File: lang/po/Firmware_cs.po (hunk 5 above)
Comment (DRracer) [reply lang/po/Firmware_cs.po]: nope, Kladka means Pulley ... but this is obviously a copy-paste error

File: lang/po/Firmware_cs.po (hunk 3 above)
Comment (DRracer) [reply lang/po/Firmware_cs.po:935]: I know, we'll figure something out

File: lang/po/Firmware_cs.po (hunk 6 above)
Comment (DRracer) [reply lang/po/Firmware_cs.po]: should read "Vysunuti ke kladce" if we keep the Pulley->Kladka translation

File: lang/po/Firmware_cs.po (hunk 7 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po:1454]: Copy paste issue on my side maybe.

File: lang/po/Firmware_cs.po (hunk 5 above)
Comment (3d-gussner) [reply lang/po/Firmware_cs.po]: is then "Motor kladce se zastavil.
correct?

Comment (3d-gussner) [reply lang/po/Firmware_cs.po]: Deepl translation
`Kladkový motor se zastavil. Zkontrolujte, zda se kladka může pohybovat, a zkontrolujte zapojení.`

**Pull request merged**